computations/x1_16_chabauty.m

```diff
@@ -1,14 +1,22 @@
+/*
+  This carries out the algorithm of Section 4.2 of the paper
+*/
+
+// First we define some standard objects
 QQ := Rationals();
 Qx<x> := PolynomialRing(QQ);
 P1:=ProjectiveSpace(QQ,1);
 
+// Innocent helper function
 function Zip(a, b)
   //similar to pythons zip function
   assert #a eq #b;
   n := #a;
   return [<a[i],b[i]> : i in [1..n]];
 end function;
 
+// This helper function helps us represent elements of the Two selmer set of the curve
```

Comment:
The wording could be clearer, first of all it is about the "fake two selmer set" not the two selmer set. Secondly this function doesn't help us represent the elements of the two selmer set. It is actually the function that is returned by this function that does this. I propose something like:

Magma's TwoCoverDescent function returns a function AtoHk that maps elements from some algebra A to the fake 2-Selmer set Hk. This function returns the inverse of this map. Elements in the algebra A can be used to construct the two-cover of the corresponding element in the fake 2-Selmer set.

Reply:
done